lib/tasks/evm_dba.rake

```diff
@@ -195,9 +179,10 @@ namespace :evm do
 
         puts "Initializing region and database..."
         # Create the region from our REGION file, initialize a new miq_database row for this region
-        EvmDba.run_cmd_with_open4("bin/rails runner 'MiqDatabase.seed; MiqRegion.seed;'")
+        AwesomeSpawn.run!("bin/rails runner", :params => ["MiqDatabase.seed; MiqRegion.seed"])
       rescue => err
-        STDERR.puts "Encountered issue setting up Database using region #{region}: #{err}\n"
+        message = err.kind_of?(AwesomeSpawn::CommandResultError) ? err.error : err.message
```

Comment:
> There could be potentially unsafe information

That sounds like a UI level concern to me... applying some filtering based on known exception classes, perhaps? It doesn't seem safe for our UI to assume `message` is "user-safe", and it doesn't seem fair for a utility gem to suppress useful info on the off-chance a less privileged user might see it.